pylint/test/unittest_checker_typecheck.py

```diff
@@ -229,3 +229,19 @@ def test_typing_namedtuple_unsubscriptable_object_issue1295(self):
         subscript = module.body[-1].value
         with self.assertNoMessages():
             self.checker.visit_subscript(subscript)
+
+    def test_staticmethod_multiprocessing_call(self):
```

Comment:
Can we also add a test with a user-defined descriptor?

Reply:
I actually cannot repro elsewhere. descriptors become `Uninferable` anywhere else due to (https://github.com/PyCQA/astroid/blob/master/astroid/scoped_nodes.py#L2333). Somehow the multiprocessing staticmethod descriptor is getting through with all the magic going on.